conf/webpack.conf.js

```diff
@@ -14,36 +18,39 @@ module.exports = {
         filename: '[name].bundle.js',
         path: path.join(__dirname, '..', 'public', 'build'),
     },
+    devtool: 'source-map',
     module: {
-        loaders: [
+        rules: [
             {
                 test:    /\.js$/,
                 exclude: /node_modules/,
-                loaders: ['babel-loader']
+                use: ['babel-loader']
             },
             {
               test: /\.(html|svg)$/,
               exclude: '/app/',
               loader: 'html-loader'
             },
-            {
-              test: /\.json$/,
-              exclude: /node_modules/,
-              use: 'json-loader'
-            },
             {
                 test: /\.scss$/,
```

Comment:
By changing this test to /\.s?css$/, can we avoid two CSS rules?

Reply:
Yeah, let's scrap that extra ruke.